### DISCUSSION TITLE
Own MX-Record with underlying Mailbox server

### DISCUSSION THREAD

**Discussion by seemedFilm** [simple-login/app#1086] [Q&A] [answered]
Hi folks,

I've used right now my free premium trial, but I did not figured out how I can archive my requirement.
I have a custom domain. This domain is currently with the MX-Record linked to mailbox.org where I can recieve and send mails with my domain.

If I change now my mx record to simple-login, where I can recieve mails, what happen when I send a new mail from mailbox.org? Or better, how can I now send mails with me@mydomain.de?

Edit: can be closed, found after 1hour looking the setting.

**Comment by nguyenkims** [comment, marked as answer]
@sectorchan2 a domain can only point to a single email service so your domain can't point to both SimpleLogin and mailbox.org. I'd suggest to use a subdomain for SimpleLogin instead.
